pallets/parachain-staking/src/lib.rs

```diff
@@ -1375,6 +1386,7 @@ pub mod pallet {
 		AlreadyDelegatedCandidate,
 		InvalidSchedule,
 		CannotSetBelowMin,
+		RoundLengthMustBeGreaterThanTotalSelectedCollators,
```

Comment:
```suggestion
		RoundLengthMustBeAtLeastTotalSelectedCollators,
```
It's greater than **or equal to** in the code now